README.md

```diff
@@ -212,13 +220,28 @@ Once the services are uninstalled, you may re-install them by running `./install
 
 *IMPORTANT*: Running the uninstall script will *NOT* remove any user data, so you can freely uninstall and re-install the services.
 
+## Migrating Package Artifacts to Minio
```

Comment:
I think we only support Minio for now since the entire premise (ha!) of the feature is that packages be stored inside of a local infrastructure and not the cloud.  If there is a specific ask, we can revisit.